packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -7020,6 +7025,13 @@ export interface IStackTokens {
     padding?: number | string;
 }
 
+// @public (undocumented)
+export interface IStickyContainerBehavior {
+    arrangeStickiesBasedOnOrder: boolean;
+    containerBehavior: StickyContainerBehaviorType;
+    notUsePlaceHolder: boolean;

```

Comment:
Consider renaming this field, it is difficult to understand. `usePlaceholder` perhaps? Or `disablePlaceholder`.